Firestore/Example/ProtobufCpp.podspec

```diff
@@ -33,7 +33,9 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = '10.13'
   s.tvos.deployment_target = '12.0'
 
-  s.source_files = 'src/**/*.{h,cc,inc}'
+  s.source_files = 'src/**/*.{h,cc,inc}',
+                   # utf8_range is not needed, and is giving out build errors.
+                   'third_party/utf8_range/*.{h,cc,inc}'
```

Comment:
The comment is confusing to me. The comment says "utf8_range is not needed", and the next line adds `utf8_range` ot the `s.source_files`!

Reply:
Done.